File: Ejercicio2.py
'''
Created on 16 nov. 2018
2.- Haz un programa en Java y Python que calcule el elemento N (se pide al usuario) de la serie de Fibonacci.

@author: d18sisaj
'''
fib=1 
fibant=0 
aux=0
posicion = int(input("Introduce la posicion de la sucecison de Fibonacci deseada: "))

for i in range(posicion-1):
    aux=fib
    fib=fib+fibant
    fibant=aux
    
print("El  numero en la posicion ",posicion," de la sucesion de Fibonacci es:  ",fib) 

File: Ejercicio2.java
package examen;
/**
 * Haz un programa en Java y Python que calcule el elemento N (se pide al usuario) de la serie de Fibonacci
 * @author d18sisaj
 *
 */
import java.util.Scanner;
public class Ejercicio2 {

  public static void main(String[] args) {
    int fib=1, fibant=0, aux=0, posicion;
    Scanner s = new Scanner(System.in);
    System.out.println("Introduzca la posicion que del numero que desea saber en la serie de Fibonacci: ");
    posicion=s.nextInt();
    
    for(int i=0;i<posicion-1; i++) {
      aux=fib;
      fib=fib+fibant;
      fibant=aux;
      
    }
    System.out.println("El numero en la posición "+posicion+" de la serie de Fibonnaci es: "+fib );
    
  }

}
